docs/README.md

```diff
@@ -22,16 +27,25 @@ Each service on this full list of services included in the repository has a desi
 - [onboarding-skill](markdown/onboarding-skill.md)
 - [storage-adapter-mongodb](markdown/storage-adapter-mongodb.md)
 
+### Architecture
+
+The following shows the services in a [TAM](http://www.fmc-modeling.org/fmc-and-tam) block diagram.
+
+![The big picture](images/AAS_SERVICE_REVISED.png).
```

Comment:
Image is not displayed

Reply:
It is not displayed in the source code but in the actual link it shows for me: https://github.com/SAP/i40-aas/tree/arch-doc/docs. Is this what you meant?

Reply:
It is not displayed in the source code but in the actual link it shows for me: https://github.com/SAP/i40-aas/tree/arch-doc/docs. Is this what you meant?

Reply:
ok when I click on the link I get a 404 from Github. Maybe a bug in the UI of Github itself.

Reply:
I am not sure what changes to perform here.